src/main/scala/com/github/dotj/hellourld/ShortLinkDto.scala

```diff
@@ -0,0 +1,7 @@
+package com.github.dotj.hellourld
+
+import scala.collection.immutable
+
+final case class ShortLinkDto(token: String, redirectToUrl: String)
+final case class ShortLinksDto(shortLinks: immutable.Seq[ShortLinkDto])
+final case class UpdateShortLinkRequest(newUrl: String)
```

Comment:
Appended these objects with `Dto` to indicate that they are json data transfer objects - though they are also used as internal objects. Ideally, I'd want to separate the data transfer objects with the core domain objects.